requirements/base.txt

```diff
@@ -3,19 +3,15 @@
 # needle (selenium is also used directly)
 nose==1.3.7
 Pillow==3.3.0
-
-# Note that to get setup.py to work correctly with needle we have
-# version constraints specified in setup.py as well. We will need
-# to change both to upgrade to selenium 3 or above.
-selenium==2.53.6
+selenium==3.3.1
 
 # Then the direct dependencies
 
 # For lazy evaluation of the page object's a11y_audit attribute
 lazy==1.2
 
 # For test assertions involving screenshots and visual diffs
-needle==0.4.1
+git+https://github.com/bfirsh/needle.git@2bc898654c5dc887d47a3ba26be92a6d42113440#egg=needle
```

Comment:
The official needle release should be out [any time now](https://github.com/bfirsh/needle/pull/70#issuecomment-289307058), so I'm ok with holding off on merging this until that's available.  I just needed something checked into a branch so I could refer to it in requirements files for local testing (which is currently in progress), and didn't want to hold up code review for it.